ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -91,6 +98,30 @@ public DistributingExecutor(
         Objects.requireNonNull(commandRunnerWarning, "commandRunnerWarning");
   }
 
+  private Optional<SourceName> shouldGeneratePlan(final KsqlExecutionContext executionContext,
```

Comment:
Rename the method for a better meaning now it belongs to the DistributingExecutor. Perhaps `getCreateSourceNameIfExists()`?